### PR TITLE
link to security related information

### PR DESCRIPTION
**Pull request by abebeos**
## Why this should be merged

Point to up-to-date security related information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,4 +245,5 @@ To officially support a new platform, one must satisfy the following requirement
 
 **We and our community welcome responsible disclosures.**
 
-If you've discovered a security vulnerability, please report it via our [bug bounty program](https://hackenproof.com/avalanche/). Valid reports will be eligible for a reward (terms and conditions apply).
+Please refer to our [Security Policy](SECURITY.md) and [Security Advisories](https://github.com/ava-labs/avalanchego/security/advisories).
+
```
Comment (StephenButtolph) [inline README.md:249]: Super minor whitespace nit:
```suggestion
```

**Comment by StephenButtolph** [comment]
This change has been staged for `v1.9.9`. Thank you!

**Comment by abebeos** [comment]
> This change has been staged for `v1.9.9`. Thank you!

How can I follow this process?

**Comment by abebeos** [comment]
> > This change has been staged for `v1.9.9`. Thank you!
> 
> How can I follow this process?

- https://github.com/ava-labs/avalanchego/issues/1212

**Comment by StephenButtolph** [comment]
The commit can be found here: https://github.com/ava-labs/avalanchego/commit/96892537d91f33db10d903087357ab5f27130451 (currently in the [dev](https://github.com/ava-labs/avalanchego/tree/dev) branch)

**Pull request closed without merging**